### DISCUSSION TITLE
Read user preferences via script

### DISCUSSION THREAD

**Discussion by doctomlug** [freeplane/freeplane#2035] [Help] [answered]
Hello,
is there any way to access (read only) the user preference values via groovy script. 
I am particularly interested in the relative/absolute path setting to be able to calculate paths correctly.
Ty

**Comment by i-plasm** [comment, marked as answer]
You may use: `org.freeplane.core.resources.ResourceController.getResourceController().getProperty("links")`

**Reply by doctomlug** [reply]
Very cool, works perfectly. Thank you!
